nilearn/interfaces/fmriprep/load_confounds.py

```diff
@@ -295,7 +304,30 @@ def load_confounds(
 
     """
     _check_strategy(strategy)
-
+    if fd_threshold == 0.2:
+        fd_threshold_default = (
+            "The default parameter for standardize is currently 0.2 "
+            "which is inconsistent with the fMRIPrep default of 0.5. "
+            "In release X, the default strategy will be replaced by "
+            "0.5."
+        )
+        warnings.warn(
+            category=DeprecationWarning,
+            message=fd_threshold_default,
+            stacklevel=3,
```

Comment:
Wonder if the stack level should not be 2 here so that the warning appears with where `load_confounds` was called.

Note: the warning stacklevel in the code base are overall wildly inconsistent, probably something we want to clean up a bit to give user better feedback.

Reply:
I would say don't sweat it too much for now until we got some rule of thumb we agree on regarding how to handle this.